app/uk/gov/hmrc/gform/commons/MarkDownUtil.scala

```diff
@@ -54,7 +54,7 @@ object MarkDownUtil {
   def markDownParser(ls: LocalisedString)(implicit l: LangADT): Html = markDownParser(ls.value)
 
   def markDownParser(ls: SmartString)(implicit sse: SmartStringEvaluator): Html = {
-    val markDownText = removeSpacesForStrong(ls.valueForMarkdown())
+    val markDownText = removeSpacesForStrong(ls.value())
```

Comment:
`ls.value`treats smart string as as html. If I leave it `ls.valueForMarkdown` all html will be escaped and will be rendered in a browser literally.

Reply:
> will be escaped and will be rendered in a browser literally

That is by design and it needs to keep behaving like that. The reason for that is that user provided input cannot be rendered as html, otherwise users would be able embed javascript into their input which will be executed etc... we don't want that.